packages/@expo/metro-config/src/transform-plugins/__tests__/import-export-plugin.test.ts

```diff
@@ -354,13 +353,9 @@ it('places export all with live bindings above export default and named', () =>
         }
       });
     });
+    var _bar = require('bar');
+    var _default = bax;
```

Comment:
is this an issue in the test? Can't see the definition for `bax`

Reply:
Yes, it's bax is also not defined in the [input code](https://github.com/expo/expo/pull/39013/files/7e2d1604e85bab38f4cdb96db27c87f541230c64#diff-04a755f866444ff364aca6ed464c132518c6574047f0d8c174a6fe52a22f5ccfR328). I'll update it, this makes it harder to know if its the plugin or the test mistake.